### PR TITLE
Adding popovers

### LINKED ISSUE #17: Add popovers for notes of a given package on the edit menu
Also add rates to this page if they are not there already

### PR DESCRIPTION
**Pull request by dandeduck**
It's derived from view form separation branch, so it will need to back merge with it before merging with dev. (it shouldn't cause conflicts tho)

### CODE REVIEW

**Comment by dandeduck** [comment]
I guess I am saying that this branch should be merged with dev after the view form separation

**Comment by dandeduck** [comment]
This branch was merged with dev in the last commit, thus making it ready for review. @tomtzook

File: templates/packages/order_edit.html [hunk 1]
```diff
@@ -36,7 +34,14 @@ <h6 class="text-danger">*קיימים שינויים לא שמורים</h6>
             <div class="col-xxl-3 col-xl-4 col-lg-4 col-md-6 col-sm-6 mb-3">
                 <div class="card h-100">
                     <div class="card-body text-start">
-                        <h5 class="card-title">{{package.destination}}</h5>
+                        <div class="row">
+                            <div class="col-auto text-decoration-none">
+                                <a type="button" data-bs-toggle="popover" data-bs-placement="bottom" data-bs-content="הערות: {{package.notes|default:'<אין>'}} מחיר: {{package.rate}} ₪"><h5 class="bi-three-dots"></h5></a>
```
Comment (tomtzook) [inline templates/packages/order_edit.html]: The idea is nice, but it doesn't seem to accommodate long comments well. It looks... cramped

popover

**Review by tomtzook — DISMISSED** [review]
Nice idea

**Review by tomtzook — CHANGES_REQUESTED** [review]
I accidentally clicked the wrong option

File: templates/packages/order_edit.html (hunk 1 above)
Comment (dandeduck) [reply templates/packages/order_edit.html]: It's not actually critical to look well, it won't be looked at much anyway. Do you have a better idea for it?

Comment (tomtzook) [reply templates/packages/order_edit.html]: You could put the price on the card and not in the _popover_

**Review by tomtzook — APPROVED** [review]

File: templates/packages/order_edit.html (hunk 1 above)
Comment (tomtzook) [reply templates/packages/order_edit.html]: If you want to make the changes, go ahead. If not, I've approved the PR.

Comment (dandeduck) [reply templates/packages/order_edit.html]: Yeah I'll put the price on the card itself

Comment (dandeduck) [reply templates/packages/order_edit.html]: Man idk where to put it

Comment (dandeduck) [reply templates/packages/order_edit.html]: @tomtzook  What do you think?

Comment (dandeduck) [reply templates/packages/order_edit.html]: > 
> 
> @tomtzook What do you think?

About where I placed it

Comment (tomtzook) [reply templates/packages/order_edit.html]: Yeah, looks good.

**Comment by tomtzook** [comment]
@dandeduck Go ahead and merge and you want.

**Pull request merged**